e2e-tests/monitoring-2-0/run

```diff
@@ -13,13 +13,19 @@ deploy_operator
 
 desc 'install PMM Server'
 platform=kubernetes
-if [ "$OPENSHIFT" == 1 ]; then
-    platform=openshift
-fi
+
 helm del --purge monitoring || :
 helm repo remove stable || :
 helm repo add stable https://charts.helm.sh/stable
-retry 10 60 helm install monitoring --set platform=$platform https://percona-charts.storage.googleapis.com/pmm-server-2.12.0.tgz
+if [[ ! -z "${OPENSHIFT}" ]]; then
+    platform=openshift
+    oc adm policy add-scc-to-user privileged -z percona-server-mongodb-operator
```

Comment:
I think we need to rollback these changes at the end of test.

Reply:
done